sds_data_manager/constructs/batch_compute_resources.py

```diff
@@ -63,7 +61,7 @@ def __init__(
             Dependent on Docker image contents.
         job_memory : int: Optional
             Memory required per Batch job in MB. Dependent on Docker image contents.
-        efs: efs.Filesystem
+        efs_instance: efs.Filesystem
```

Comment:
Thank you for catching this.
```suggestion
        efs_instance: EFSStack
```